addons/web_calendar/static/src/js/web_calendar.js

```diff
@@ -21,13 +21,23 @@ var QWeb = core.qweb;
 function get_fc_defaultOptions() {
     var dateFormat = time.strftime_to_moment_format(_t.database.parameters.date_format);
 
+    // moment.js converts '%p' to 'A' for 'AM/PM'
+    // But FullCalendar v1.6.4 supports 'TT' format for 'AM/PM' but not 'A'
```

Comment:
The date time format strings will change - so we must adapt this then. But for fullcalendar 2.0 i think this will be a nearly complete rewrite - so this patch would be perfect in current v9.0c